frontend/src/pages/new_document.tsx

```diff
@@ -172,7 +179,9 @@ export function NewDocumentPage() {
               <SecondaryButton type="button" onClick={() => navigate(`/`)}>
                 Cancel
               </SecondaryButton>
-              <PrimaryButton type="submit">Create</PrimaryButton>
+              <PrimaryButton disabled={loading} type="submit">
+                {loading ? <LoadingSpinner /> : 'Create'}
+              </PrimaryButton>
```

Comment:
Is it possible to have the spinner have the same size as the button?

Reply:
Done